pettingzoo/utils/wrappers/base.py

```diff
@@ -39,6 +39,12 @@ def __init__(self, env):
         except AttributeError:
             pass
 
+    def __getattr__(self, name):
+        """Returns an attribute with ``name``, unless ``name`` starts with an underscore."""
+        if name.startswith("_"):
+            raise AttributeError(f"accessing private attribute '{name}' is prohibited")
+        return getattr(self.env, name)
+
```

Comment:
@benblack769 for reference

Reply:
I wouldn't add it casually in the middle of a big PR like this, its worth thinking over whether this plays nicely with the conversion wrappers, with the parallel wrappers, with the OrderEnforcing wrapper which has its own `__getattr__` overload, etc.

PZ has somewhat more complex wrapper system than gym.

That being said, I think people have converted me over to the concept that this is a good idea.